app/views/users/index.html.erb

```diff
@@ -1 +1,13 @@
-<p>Here is a list of all the users:</p>
\ No newline at end of file
+<div class='index'>
+  <% @users.each do |user| %>
```

Comment:
- [OPTIONAL] Although this is good, it would be better to  have a partial `_user.html.erb` located in `app/views/users/_user.html.erb` that you can reuse as a template for each user in your each block and render it in your index.html.erb just by using the tag
`<%= render @users %>`